virtualization/windowscontainers/container-networking/advanced.md

```diff
@@ -96,15 +90,26 @@ We're working on platform changes to automatically detect/prevent this issue. Cu
 C:\> reg delete HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\Tcpip6\Parameters  /v DisabledComponents  /f
 ```
 
+
+#### Linux Containers on Windows
+
+**NEW:** We are working to make it possible to run Linux and Windows containers side-by-side _without the Moby Linux VM_. See this [blog post about Linux Containers on Windows (LCOW)](https://blog.docker.com/2017/11/docker-for-windows-17-11/) for details.
+
 #### Moby Linux VMs use DockerNAT switch with Docker for Windows (a product of [Docker CE](https://www.docker.com/community-edition)) instead of HNS internal vSwitch 
+
 Docker for Windows (the Windows driver for the Docker CE engine) on Windows 10 will use an Internal vSwitch named 'DockerNAT' to connect Moby Linux VMs to the container host. Developers using Moby Linux VMs on Windows should be aware that their hosts are using the DockerNAT vSwitch rather than the vSwitch that is created by the HNS service (which is the default switch used for Windows containers).
 
-#### To use DHCP for IP assignment on a virtual container host enable MACAddressSpoofing 
+
+#### To use DHCP for IP assignment on a virtual container host enable MACAddressSpoofing
+>  _MAC address spoofing is required for l2bridge and transparent network drivers **as well as Kubernetes**_
```

Comment:
Remove l2bridge from this section as it does'nt make sense in this context. Instead, add a note in the l2bridge and l2tunnel network driver section that "MAC Address spoofing" or "promiscuous mode" is required to be set in the virtualiztion host for the VM Network Adapter attached to the Container host.